src/consola.ts

```diff
@@ -21,6 +21,8 @@ const queue: any[] = [];
 export class Consola {
   options: ConsolaOptions;
 
+  private groupIndentionLevel = 0;
```

Comment:
I am thinking instead of having an integer, we can store group titles, for example, `consola.group('building..')` can be displayed (or used for interceptors)

Reply:
The `groupIndentionLevel` tracks the indentation level since `.group` can be nested.

If we consider `.group` as a log type, the current logic can display group titles correctly without further changes.